Gemfile

```diff
@@ -2,36 +2,32 @@
 
 source 'https://rubygems.org'
 
+gem 'dotenv-rails', require: 'dotenv/rails-now'
+
 git_source(:github) do |repo_name|
   repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?('/')
   "https://github.com/#{repo_name}.git"
 end
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.1.4'
-# Use sqlite3 as the database for Active Record
-gem 'sqlite3'
 # Use Puma as the app server
 gem 'puma', '~> 3.7'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.5'
-# Use Redis adapter to run Action Cable in production
-# gem 'redis', '~> 3.0'
 # Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
-
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
+gem 'bcrypt', '~> 3.1.7'
 
-# Use Rack CORS for handling Cross-Origin Resource Sharing (CORS),
-# making cross-origin AJAX possible
-# gem 'rack-cors'
+group :production do
+  gem 'pg', '~> 0.21'
+  gem 'rails_12factor'
+end
 
 group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger
-  # console
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'factory_girl_rails', '~> 4.8'
   gem 'rspec-rails', '~> 3.6'
+  gem 'sqlite3'
```

Comment:
just note: for simple applications it's ok to use sqlite for development, but keep in mind that for real projects better to use ONE db backend (no matter what env is: dev, test, or production), for ex: postgresql

Reply:
changed to postgresql